src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,102 @@
+@import '../variables';
+
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 32px;
+  height: auto;
+  background-color: $background-color;
+  border: 1px solid $border-color;
+  color: $text-color;
+  box-sizing: border-box;
+
+  //&:hover { background-color: $background-color; }//
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__text-container {
+    text-align: left;
+  }
+
+  &__title {
+    font-family: $font-primary;
+    font-size: 12px;
+    font-weight: 500;
+    margin: 0 16px 4px;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $secondary-text-color;
+    line-height: 14px;
+    margin: 4px 16px 16px;
+    font-family: $font-primary;
+    font-weight: 400;
+  }
+  &__price-container {
+    justify-content: space-between;
+    align-items: center;
+    display: flex;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: calc(100% - 32px);
+
+    &-label {
+      font-family: $font-primary;
+      font-size: 12px;
+      font-weight: 400;
+      color: $secondary-text-color;
+      line-height: 18px;
+      text-align: left;
+    }
+
+    &-value {
+      font-family: $font-primary;
+      font-size: 16px;
+      font-weight: bold;
+      color: $text-color;
+      line-height: 18px;
+      text-align: right;
+      margin-left: 85px;
+    }
+  }
+
+  &__buy {
+    font-family: $font-primary;
+    margin: 16px;
+    display: flex;
+    width: calc(100% - 32px);
+    justify-content: center;
+    align-items: center;
+    background-color: $primary-color;
+    color: $background-color;
+    font-size: 14px;
+    font-weight: 700;
+    width: 166px;
```

Comment:
The `width` property is defined twice in the `&__buy` class. Consider removing one of the definitions to avoid redundancy and potential conflicts. Choose the appropriate width value that fits your design requirements.